app/views/hyrax/assets/_representative_media.html.erb

```diff
@@ -1,16 +1,20 @@
-<%
-# This is an override of Hyrax 2.1.0 [master] [samvera-labs/hyrax-iiif_av/blob/master/app/views/hyrax/base/_representative_media.html.erb]
-# WARNING: upgrading Hyrax may break this override.
-# Gets default thumbnail from solr document instead of using default.png
-%>
+<% media_type = presenter.solr_document['media_type_ssim'].first %>
+<% if ["Moving Image", "Audio"].include? media_type %>
+  <% content_tag_type, source_type = media_type == "Moving Image" ? ["video", "video/mp4"] : ["audio", "audio/mp3"] %>
 
-<%# if presenter.representative_id.present? && presenter.representative_presenter.present?%>
-<% if true %>
-  <% if defined?(viewer) && viewer %>
-    <%= iiif_viewer_display presenter %>
-  <% else %>
-    <%= media_display presenter.representative_presenter %>
-  <% end %>
+
+ <%= content_tag(content_tag_type,
+              controls: true,
+              "class" => "video-js vjs-default-skin vjs-fluid",
+              "id" => "player_media",
+              "aria-label" => "video player",
+              oncontextmenu: 'return false;',
+              preload: 'auto',
+              crossorigin: 'anonymous',
+              :"data-setup" => '{}',
+              poster: 's') do %>
+                <source src="/media/<%= presenter.id %>" type='<%= source_type %>' />
+              <% end %>
 <% else %>
-  <%= image_tag "#{presenter.solr_document.fetch('thumbnail_path_ss')}", class: "canonical-image" %>
-<% end %>
\ No newline at end of file
+  Not playable
```

Comment:
I just thew "Not playable" in there as a placeholder. Can include the media type to provide more of a clue?
```
Not playable: unrecognized media type: "<%= media type %>".
```